dateutil/rrule.py

```diff
@@ -707,7 +707,7 @@ def __str__(self):
             parts.append('INTERVAL=' + str(self._interval))
 
         if self._wkst:
-            parts.append('WKST=' + str(self._wkst))
+            parts.append('WKST=' + repr(weekday(self._wkst))[0:2])
```

Comment:
Just idle speculation, I'm wondering if maybe `str(weekday)` should return `MO` for `MO(0)`. Then we could just replace this with `str(weekday(self._wkst(0))))`.

I think that's a matter for a different PR, though.